mvc/controllers.js

```diff
@@ -32,3 +34,13 @@ exports.getArticleByIdController = (req, res, next) => {
         })
         .catch(next)
 }
+
+exports.getCommentsByArticleIdController = (req, res, next) => {
```

Comment:
up to you, but the function doesn't need the name controller included if it's already in a controller file :)